README.md

```diff
@@ -1 +1,12 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+This is the Hello World project for my Microverse training.
+
+In this project I learned couple things as stated below.
+
+- Creating gitignore file and use it for node modules
+- Installing linters and using them
+- Creating index.html file with a h1 tag
+- Creating css file to design h1 tag
+- Creating test.md file and ignoring it with the use of gitignore file
+- Opening a pull request
```

Comment:
- [ ] Also, for the readme file we need to make some adjustments to make it even better. 1. You should use a descriptive README AKA: [Microverse README template](https://github.com/microverseinc/readme-template).

Reply:
Tadesse, I managed to style it better. On the other hand, you don't expect me to put the Microverse README template for my project, right?

Reply:
Dear @mertkantaroglu, Sorry for the late reply, But please use the template and modify the contents to your project information. If there is no information to put in some of those heading leave them empty. It's always advised to use the template.

Reply:
Thanks Tadesse. I am fixing it now.